### DISCUSSION TITLE
"Aw, Snap!" errors (mainly Status_Access_Violation) on Google Chrome, and the rare BSOD

### DISCUSSION THREAD

**Discussion by SomeNYBoi1098** [seerge/g-helper#712] [General]
Ever since the update that introduced Undervolting released and I messed around with the new settings for a few minutes to see what temps I could get, I've been getting "Aw, Snap!" errors on Google Chrome, and once in a while, blue screens any time that GHelper is in use. I've tried reinstalling GHelper, deleting my config file, everything, the only thing that helps is literally just NOT having GHelper running, and keep in mind, these errors happen with NO settings in GHelper checked or changed.

A few minutes ago I've tried to open GHelper again after not using it for about 3 days, keep in mind I haven't seen ANY errors in that time, but it only took mere minutes to receive a Status_Access_Violation error. I'm sure this is a GHelper issue because even now, after I closed GHelper and am now writing this, I have not gotten and error since. 

I'm on a 2022 G14 with a RX 6700S, using v0.91 of GHelper, if I can't fix this soon, I'm just gonna go back to Armory Crate.

**Comment by seerge** [comment]
@SomeNYBoi1098 Hello, if you have undervolted too much you could cause cause cpu instability (as it's pretty much says in a big red warning box) :). Since you don’t provide neither logs nor configs from ``%appdata%/ghelper`` by some reason i can’t really check anything more :)

My general advice would be upgrade to latest 0.93 and set /apply UV values to 0, and uncheck auto apply :) 

Generally speaking UV is being reset after rebooting your laptop.

**Reply by seerge** [reply]
P.S. You may also try a [hardware reset](https://github.com/seerge/g-helper#how-do-i-do-a-hardware-reset-on-a-laptop) :)

**Reply by SomeNYBoi1098** [reply]
Tried both of these, didn't work, got a Aw Snap error, a few minutes later Chrome closed on its own, and minutes later my laptop restarted.

**Reply by seerge** [reply]
Can you post a log / config here ?

**Reply by SomeNYBoi1098** [reply]
Config:

{
  "performance_mode": 2,
  "performance_1": 2,
  "gpu_mode": 0,
  "internal_display": "TL140ADXP01",
  "frequency": 120,
  "overdrive": 0,
  "mid_fan": 0,
  "xgm_fan": 0,
  "fan_profile_cpu_0": "14-3A-3D-40-48-4D-51-62-00-08-11-16-22-29-30-45",
  "fan_profile_gpu_0": "14-3A-3D-40-48-4D-51-62-00-0C-16-1D-26-2D-34-4A",
  "limit_total_0": 125,
  "limit_cpu_0": 80,
  "limit_fast_0": 125,
  "fan_profile_cpu_2": "14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D",
  "fan_profile_gpu_2": "14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D",
  "limit_total_2": 106,
  "limit_cpu_2": 70,
  "limit_fast_2": 125,
  "gpu_auto": 0,
  "fan_rpm": 0,
  "auto_apply_2": 0,
  "auto_uv_2": 1,
  "cpu_temp_2": 96,
  "cpu_uv_2": 0,
  "igpu_uv_2": 0,
  "performance_0": 2,
  "charge_limit": 100,
  "auto_apply_power_2": 0
}

Log:

7/2/2023 7:59:42 AM: ------------
7/2/2023 7:59:42 AM: App launched: ROG Zephyrus G14 GA402RJ_GA402RJ :0.91.0.0en-US
7/2/2023 7:59:43 AM: CPU: 8 - AMD Ryzen 9 6900HS with Radeon Graphics         - AMD64 Family 25 Model 68 Stepping 1
7/2/2023 7:59:43 AM: AutoSetting for Online
7/2/2023 7:59:43 AM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 7:59:43 AM: Mode = 0 : OK
7/2/2023 7:59:43 AM: Eco flag : 0
7/2/2023 7:59:43 AM: Mux flag : 1
7/2/2023 7:59:43 AM: Auto:5D-BA-C5-C4-01
7/2/2023 7:59:43 AM: Monitor Power On
7/2/2023 7:59:49 AM: Mode = 2 : OK
7/2/2023 7:59:50 AM: 
7/2/2023 7:59:50 AM: Stopped: RadeonSoftware
7/2/2023 7:59:50 AM: Stopped: EpicGamesLauncher
7/2/2023 7:59:50 AM: Stopped: ASUSSmartDisplayControl
7/2/2023 7:59:50 AM: Running eco command 1
7/2/2023 7:59:58 AM: GPUEco = 1 : OK
7/2/2023 7:59:58 AM: Eco flag : 1
7/2/2023 7:59:58 AM: Mux flag : 1
7/2/2023 7:59:58 AM: ScreenOverdrive = 0 : OK
7/2/2023 8:00:56 AM: Mode = 2 : OK
7/2/2023 8:00:56 AM: FanCPU = 14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D : OK
7/2/2023 8:00:56 AM: FanGPU = 14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D : OK
7/2/2023 9:06:01 AM: ------------
7/2/2023 9:06:01 AM: App launched: ROG Zephyrus G14 GA402RJ_GA402RJ :0.91.0.0en-US
7/2/2023 9:06:03 AM: CPU: 8 - AMD Ryzen 9 6900HS with Radeon Graphics         - AMD64 Family 25 Model 68 Stepping 1
7/2/2023 9:06:03 AM: AutoSetting for Online
7/2/2023 9:06:03 AM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 9:06:03 AM: Mode = 2 : OK
7/2/2023 9:06:03 AM: FanCPU = 14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D : OK
7/2/2023 9:06:03 AM: FanGPU = 14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D : OK
7/2/2023 9:06:03 AM: Eco flag : 1
7/2/2023 9:06:03 AM: Mux flag : 1
7/2/2023 9:06:03 AM: ScreenOverdrive = 0 : OK
7/2/2023 9:06:03 AM: Auto:5D-BA-C5-C4-01
7/2/2023 9:06:03 AM: Monitor Power On
7/2/2023 9:06:17 AM: ------------
7/2/2023 9:06:17 AM: App launched: ROG Zephyrus G14 GA402RJ_GA402RJ :0.91.0.0en-US.
7/2/2023 9:06:19 AM: CPU: 8 - AMD Ryzen 9 6900HS with Radeon Graphics         - AMD64 Family 25 Model 68 Stepping 1
7/2/2023 9:06:19 AM: AutoSetting for Online
7/2/2023 9:06:19 AM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 9:06:19 AM: Mode = 2 : OK
7/2/2023 9:06:19 AM: FanCPU = 14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D : OK
7/2/2023 9:06:19 AM: FanGPU = 14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D : OK
7/2/2023 9:06:19 AM: Eco flag : 1
7/2/2023 9:06:19 AM: Mux flag : 1
7/2/2023 9:06:19 AM: ScreenOverdrive = 0 : OK
7/2/2023 9:06:19 AM: Auto:5D-BA-C5-C4-01
7/2/2023 9:06:19 AM: UV: 0 OK
7/2/2023 9:06:19 AM: iGPU UV: 0 OK
7/2/2023 9:06:19 AM: CPU Temp: 96 OK
7/2/2023 9:06:19 AM: APU Temp: 96 OK
7/2/2023 9:06:19 AM: Monitor Power On
7/2/2023 9:06:23 AM: UV: 0 OK
7/2/2023 9:06:24 AM: iGPU UV: 0 OK
7/2/2023 9:06:24 AM: CPU Temp: 96 OK
7/2/2023 9:06:24 AM: APU Temp: 96 OK
7/2/2023 9:06:54 AM: ------------
7/2/2023 9:06:54 AM: App launched: ROG Zephyrus G14 GA402RJ_GA402RJ :0.91.0.0en-US.
7/2/2023 9:06:55 AM: CPU: 8 - AMD Ryzen 9 6900HS with Radeon Graphics         - AMD64 Family 25 Model 68 Stepping 1
7/2/2023 9:06:55 AM: AutoSetting for Offline
7/2/2023 9:06:55 AM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 9:06:55 AM: Mode = 2 : OK
7/2/2023 9:06:55 AM: FanCPU = 14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D : OK
7/2/2023 9:06:55 AM: FanGPU = 14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D : OK
7/2/2023 9:06:55 AM: Eco flag : 0
7/2/2023 9:06:55 AM: Mux flag : 1
7/2/2023 9:06:55 AM: ScreenOverdrive = 0 : OK
7/2/2023 9:06:55 AM: Monitor Power On
7/2/2023 9:06:55 AM: Auto:5D-BA-C5-C4-01
7/2/2023 9:06:56 AM: UV: 0 OK
7/2/2023 9:06:56 AM: iGPU UV: 0 OK
7/2/2023 9:06:57 AM: CPU Temp: 96 OK
7/2/2023 9:06:57 AM: APU Temp: 96 OK
7/2/2023 9:07:02 AM: 
7/2/2023 9:07:02 AM: Stopped: EpicGamesLauncher
7/2/2023 9:07:02 AM: Stopped: ASUSSmartDisplayControl
7/2/2023 9:07:02 AM: Running eco command 1
7/2/2023 9:07:10 AM: GPUEco = 1 : OK
7/2/2023 9:07:10 AM: Eco flag : 1
7/2/2023 9:07:10 AM: Mux flag : 1
7/2/2023 9:07:10 AM: ScreenOverdrive = 0 : OK
7/2/2023 9:16:16 AM: Monitor Power Off
7/2/2023 9:16:16 AM: Backlight:5D-BA-C5-C4-00
7/2/2023 9:16:16 AM: WMI event 192
7/2/2023 9:21:41 AM: Monitor Power On
7/2/2023 9:21:41 AM: AutoSetting for Offline
7/2/2023 9:21:41 AM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 9:21:41 AM: Mode = 2 : OK
7/2/2023 9:21:41 AM: FanCPU = 14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D : OK
7/2/2023 9:21:41 AM: FanGPU = 14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D : OK
7/2/2023 9:21:41 AM: Eco flag : 1
7/2/2023 9:21:41 AM: Mux flag : 1
7/2/2023 9:21:41 AM: ScreenOverdrive = 0 : OK
7/2/2023 9:21:41 AM: Auto:5D-BA-C5-C4-01
7/2/2023 9:21:42 AM: UV: 0 OK
7/2/2023 9:21:42 AM: iGPU UV: 0 OK
7/2/2023 9:21:42 AM: CPU Temp: 96 OK
7/2/2023 9:21:42 AM: APU Temp: 96 OK
7/2/2023 9:36:30 AM: Monitor Dimmed
7/2/2023 9:36:46 AM: Monitor Power Off
7/2/2023 9:36:46 AM: Backlight:5D-BA-C5-C4-00
7/2/2023 3:58:44 PM: ------------
7/2/2023 3:58:44 PM: App launched: ROG Zephyrus G14 GA402RJ_GA402RJ :0.91.0.0en-US.
7/2/2023 3:58:45 PM: CPU: 8 - AMD Ryzen 9 6900HS with Radeon Graphics         - AMD64 Family 25 Model 68 Stepping 1
7/2/2023 3:58:47 PM: AutoSetting for Offline
7/2/2023 3:58:47 PM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 3:58:47 PM: Mode = 2 : OK
7/2/2023 3:58:47 PM: FanCPU = 14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D : OK
7/2/2023 3:58:47 PM: FanGPU = 14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D : OK
7/2/2023 3:58:47 PM: Eco flag : 1
7/2/2023 3:58:47 PM: Mux flag : 1
7/2/2023 3:58:47 PM: ScreenOverdrive = 0 : OK
7/2/2023 3:58:47 PM: Monitor Power On
7/2/2023 3:58:47 PM: Auto:5D-BA-C5-C4-01
7/2/2023 3:58:48 PM: UV: 0 OK
7/2/2023 3:58:48 PM: iGPU UV: 0 OK
7/2/2023 3:58:48 PM: CPU Temp: 96 OK
7/2/2023 3:58:48 PM: APU Temp: 96 OK
7/2/2023 3:59:35 PM: BatteryLimit = 85 : OK
7/2/2023 3:59:35 PM: BatteryLimit = 90 : OK
7/2/2023 3:59:35 PM: BatteryLimit = 95 : OK
7/2/2023 3:59:35 PM: BatteryLimit = 100 : OK
7/2/2023 3:59:40 PM: Mode = 2 : OK
7/2/2023 3:59:40 PM: FanCPU = 14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D : OK
7/2/2023 3:59:40 PM: FanGPU = 14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D : OK
7/2/2023 3:59:41 PM: PowerLimit A0 = 125 : OK
7/2/2023 3:59:41 PM: PowerLimit A3 = 125 : OK
7/2/2023 3:59:41 PM: PowerLimit B0 = 80 : OK
7/2/2023 3:59:41 PM: UV: 0 OK
7/2/2023 3:59:42 PM: iGPU UV: 0 OK
7/2/2023 3:59:42 PM: CPU Temp: 96 OK
7/2/2023 3:59:42 PM: APU Temp: 96 OK
7/2/2023 3:59:46 PM: PowerLimit A0 = 125 : OK
7/2/2023 3:59:46 PM: PowerLimit A3 = 125 : OK
7/2/2023 3:59:46 PM: PowerLimit B0 = 70 : OK
7/2/2023 3:59:46 PM: UV: 0 OK
7/2/2023 3:59:46 PM: iGPU UV: 0 OK
7/2/2023 3:59:46 PM: CPU Temp: 96 OK
7/2/2023 3:59:46 PM: APU Temp: 96 OK
7/2/2023 3:59:49 PM: PowerLimit A0 = 107 : OK
7/2/2023 3:59:49 PM: PowerLimit A3 = 107 : OK
7/2/2023 3:59:49 PM: PowerLimit B0 = 70 : OK
7/2/2023 3:59:49 PM: UV: 0 OK
7/2/2023 3:59:49 PM: iGPU UV: 0 OK
7/2/2023 3:59:49 PM: CPU Temp: 96 OK
7/2/2023 3:59:49 PM: APU Temp: 96 OK
7/2/2023 3:59:52 PM: PowerLimit A0 = 107 : OK
7/2/2023 3:59:52 PM: PowerLimit A3 = 107 : OK
7/2/2023 3:59:52 PM: PowerLimit B0 = 70 : OK
7/2/2023 3:59:52 PM: UV: 0 OK
7/2/2023 3:59:52 PM: iGPU UV: 0 OK
7/2/2023 3:59:52 PM: CPU Temp: 96 OK
7/2/2023 3:59:52 PM: APU Temp: 96 OK
7/2/2023 3:59:54 PM: PowerLimit A0 = 106 : OK
7/2/2023 3:59:54 PM: PowerLimit A3 = 106 : OK
7/2/2023 3:59:54 PM: PowerLimit B0 = 70 : OK
7/2/2023 3:59:54 PM: UV: 0 OK
7/2/2023 3:59:54 PM: iGPU UV: 0 OK
7/2/2023 3:59:54 PM: CPU Temp: 96 OK
7/2/2023 3:59:54 PM: APU Temp: 96 OK
7/2/2023 4:45:27 PM: Monitor Dimmed
7/2/2023 4:45:43 PM: Monitor Power Off
7/2/2023 4:45:43 PM: Backlight:5D-BA-C5-C4-00
7/2/2023 5:00:57 PM: Monitor Power On
7/2/2023 5:00:57 PM: AutoSetting for Offline
7/2/2023 5:00:57 PM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 5:00:57 PM: BatteryLimit = 100 : OK
7/2/2023 5:00:57 PM: Mode = 2 : OK
7/2/2023 5:00:57 PM: FanCPU = 14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D : OK
7/2/2023 5:00:57 PM: FanGPU = 14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D : OK
7/2/2023 5:00:57 PM: Eco flag : 1
7/2/2023 5:00:57 PM: Mux flag : 1
7/2/2023 5:00:57 PM: ScreenOverdrive = 0 : OK
7/2/2023 5:00:57 PM: Auto:5D-BA-C5-C4-01
7/2/2023 5:00:58 PM: PowerLimit A0 = 106 : OK
7/2/2023 5:00:58 PM: PowerLimit A3 = 106 : OK
7/2/2023 5:00:58 PM: PowerLimit B0 = 70 : OK
7/2/2023 5:00:58 PM: UV: 0 OK
7/2/2023 5:00:58 PM: iGPU UV: 0 OK
7/2/2023 5:00:58 PM: CPU Temp: 96 OK
7/2/2023 5:00:58 PM: APU Temp: 96 OK
7/2/2023 5:01:14 PM: Listener stopped
7/2/2023 5:06:38 PM: ------------
7/2/2023 5:06:38 PM: App launched: ROG Zephyrus G14 GA402RJ_GA402RJ :0.91.0.0en-US.
7/2/2023 5:06:39 PM: CPU: 8 - AMD Ryzen 9 6900HS with Radeon Graphics         - AMD64 Family 25 Model 68 Stepping 1
7/2/2023 5:06:39 PM: AutoSetting for Offline
7/2/2023 5:06:39 PM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 5:06:39 PM: BatteryLimit = 100 : OK
7/2/2023 5:06:39 PM: BatteryLimit = 100 : OK
7/2/2023 5:06:39 PM: Mode = 2 : OK
7/2/2023 5:06:39 PM: FanCPU = 14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D : OK
7/2/2023 5:06:39 PM: FanGPU = 14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D : OK
7/2/2023 5:06:39 PM: Eco flag : 1
7/2/2023 5:06:39 PM: Mux flag : 1
7/2/2023 5:06:39 PM: ScreenOverdrive = 0 : OK
7/2/2023 5:06:39 PM: Monitor Power On
7/2/2023 5:06:39 PM: Auto:5D-BA-C5-C4-01
7/2/2023 5:06:40 PM: PowerLimit A0 = 106 : OK
7/2/2023 5:06:40 PM: PowerLimit A3 = 106 : OK
7/2/2023 5:06:40 PM: PowerLimit B0 = 70 : OK
7/2/2023 5:06:40 PM: UV: 0 OK
7/2/2023 5:06:40 PM: iGPU UV: 0 OK
7/2/2023 5:06:40 PM: CPU Temp: 96 OK
7/2/2023 5:06:40 PM: APU Temp: 96 OK
7/2/2023 5:06:49 PM: Mode = 2 : OK
7/2/2023 5:06:49 PM: FanCPU = 14-31-41-44-47-4C-52-62-00-1C-22-22-28-29-2A-2D : OK
7/2/2023 5:06:49 PM: FanGPU = 14-37-41-42-47-4B-4C-62-00-18-22-22-24-26-26-2D : OK
7/2/2023 5:06:50 PM: UV: 0 OK
7/2/2023 5:06:50 PM: iGPU UV: 0 OK
7/2/2023 5:06:50 PM: CPU Temp: 96 OK
7/2/2023 5:06:50 PM: APU Temp: 96 OK
7/2/2023 5:06:50 PM: Mode = 2 : OK
7/2/2023 5:06:51 PM: UV: 0 OK
7/2/2023 5:06:51 PM: iGPU UV: 0 OK
7/2/2023 5:06:51 PM: CPU Temp: 96 OK
7/2/2023 5:06:51 PM: APU Temp: 96 OK
7/2/2023 6:43:45 PM: ------------
7/2/2023 6:43:45 PM: App launched: ROG Zephyrus G14 GA402RJ_GA402RJ :0.91.0.0en-US
7/2/2023 6:43:46 PM: CPU: 8 - AMD Ryzen 9 6900HS with Radeon Graphics         - AMD64 Family 25 Model 68 Stepping 1
7/2/2023 6:43:47 PM: AutoSetting for Offline
7/2/2023 6:43:47 PM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 6:43:47 PM: BatteryLimit = 100 : OK
7/2/2023 6:43:47 PM: BatteryLimit = 100 : OK
7/2/2023 6:43:47 PM: Mode = 2 : OK
7/2/2023 6:43:47 PM: Eco flag : 1
7/2/2023 6:43:47 PM: Mux flag : 1
7/2/2023 6:43:47 PM: ScreenOverdrive = 0 : OK
7/2/2023 6:43:47 PM: Auto:5D-BA-C5-C4-01
7/2/2023 6:43:47 PM: Monitor Power On

**Reply by seerge** [reply]
@SomeNYBoi1098 oke, i see that "Auto Apply" under advanced seem to be still checked (as app applies temp / UV limits, even tho they are 0) It could be that by some reason, even 0 values cause instability in your silicone. 

```
7/2/2023 5:06:50 PM: Mode = 2 : OK
7/2/2023 5:06:51 PM: UV: 0 OK
7/2/2023 5:06:51 PM: iGPU UV: 0 OK
7/2/2023 5:06:51 PM: CPU Temp: 96 OK
7/2/2023 5:06:51 PM: APU Temp: 96 OK
```

Can you **uncheck** "Auto Apply" for all your modes (according to config it's Turbo)? And then reboot for clean test ? (just in case "factory defaults" button would also uncheck it)

And then see how your system behaves w/o g-helper running or with it. _(and post a fresh log)_

**Reply by SomeNYBoi1098** [reply]
I tried this and thought it was working, until I got a blue screen out of nowhere, the blue screen happened at about 8:25PM

Log:

7/2/2023 7:25:23 PM: ------------
7/2/2023 7:25:23 PM: App launched: ROG Zephyrus G14 GA402RJ_GA402RJ :0.93.0.0en-US.
7/2/2023 7:25:24 PM: CPU: 8 - AMD Ryzen 9 6900HS with Radeon Graphics         - AMD64 Family 25 Model 68 Stepping 1
7/2/2023 7:25:24 PM: AutoSetting for Offline
7/2/2023 7:25:24 PM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 7:25:24 PM: BatteryLimit = 100 : OK
7/2/2023 7:25:24 PM: BatteryLimit = 100 : OK
7/2/2023 7:25:24 PM: Mode = 0 : OK
7/2/2023 7:25:24 PM: Eco flag : 1
7/2/2023 7:25:24 PM: Mux flag : 1
7/2/2023 7:25:24 PM: ScreenOverdrive = 0 : OK
7/2/2023 7:25:24 PM: Monitor Power On
7/2/2023 7:25:24 PM: Auto:5D-BA-C5-C4-01
7/2/2023 7:46:20 PM: Monitor Power Off
7/2/2023 7:46:20 PM: Backlight:5D-BA-C5-C4-00
7/2/2023 8:22:09 PM: Monitor Power On
7/2/2023 8:22:09 PM: AutoSetting for Offline
7/2/2023 8:22:09 PM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 8:22:09 PM: BatteryLimit = 100 : OK
7/2/2023 8:22:09 PM: Mode = 0 : OK
7/2/2023 8:22:09 PM: Eco flag : 1
7/2/2023 8:22:09 PM: Mux flag : 1
7/2/2023 8:22:09 PM: ScreenOverdrive = 0 : OK
7/2/2023 8:22:09 PM: Auto:5D-BA-C5-C4-01
7/2/2023 8:26:13 PM: ------------
7/2/2023 8:26:13 PM: App launched: ROG Zephyrus G14 GA402RJ_GA402RJ :0.93.0.0en-US.
7/2/2023 8:26:14 PM: CPU: 8 - AMD Ryzen 9 6900HS with Radeon Graphics         - AMD64 Family 25 Model 68 Stepping 1
7/2/2023 8:26:14 PM: AutoSetting for Offline
7/2/2023 8:26:14 PM: Input: \\?\hid#vid_0b05&pid_19b6&mi_00&col01#8&18c99fbc&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
7/2/2023 8:26:14 PM: BatteryLimit = 100 : OK
7/2/2023 8:26:14 PM: BatteryLimit = 100 : OK
7/2/2023 8:26:14 PM: Mode = 0 : OK
7/2/2023 8:26:14 PM: Eco flag : 1
7/2/2023 8:26:14 PM: Mux flag : 1
7/2/2023 8:26:14 PM: ScreenOverdrive = 0 : OK
7/2/2023 8:26:14 PM: Monitor Power On
7/2/2023 8:26:14 PM: Auto:5D-BA-C5-C4-01

**Reply by seerge** [reply]
@SomeNYBoi1098 I see that on this stage (if log starts after a reboot?) app doesn't set any CPU settings on your device. It just sets a default (balanced) mode w/o any custom settings + listens for hotkeys via USB. This wasn't added or changed anyhow in last versions neither.

1. I saw that laptop was turning off screen / going to sleep in a log, mb that's a reason? Can you check in windows Even Viewer for an actual reasons of your crash? 

2. Since crash happened after waking up from sleep (?) - try to disable "Automatic windows power modes" under Extra menu in the app. It would just set a corresponding power mode each time if enabled ![power mode](https://images.idgesg.net/images/article/2021/07/windows-11-power-mode-100895295-large.jpg?auto=webp&quality=85,70) even tho it's pretty much standard thing in windows
